docs/wallet/wallet-extension.md

```diff
@@ -24,104 +22,133 @@ Let's walk through the steps how to install and set up the Maiar DeFi Wallet ext
 
 * You will be presented with two options: you can either *Create new wallet* or *Import existing wallet.*
 
-
 ![maiar-defi-example-welcomepage](/wallet/wallet-extension/wallet_extension_step4.png)
 
-* We will be creating a new wallet. Click on "*Create new wallet"*. However, if you have an existing wallet, click on *Import existing wallet*. 
-* A list of secret phrases will be displayed. Copy this phrase and agree to the instruction below.
 
-> **Note**: These secret phrases should be kept confidential and not revealed to anyone.
+### Create a new wallet
+
+**Step 1:** To get started, first install the [Maiar DeFi wallet extension](https://chrome.google.com/webstore/detail/maiar-defi-wallet/dngmlblcodfobpdpecaadgfbcggfjfnm).
+
+**Step 2:** Open up the extension and click on _‘’Create new wallet”_.
+
+**Step 3:** Next, a set of 24-secret phrases will be displayed. Safely backup these secret phrases. We strongly recommend you copy the phrases or write them down and store in a safe place like a password manager.  These phrases would be used to access your wallet and may not be retrieved.
 
 ![maiar-defi-example-secretphrase](/wallet/wallet-extension/wallet_extension_step5.png)
 
-* For security purposes, you will be prompted to input some of the secret words.
+**Step 4:** Before proceeding to the next step, confirm that you have safely stored your secret phrase.
+
+**Step 5:** For further verification, you will be prompted to input some of the secret phrases. 
 
 ![maiar-defi-example-authentication](/wallet/wallet-extension/wallet_extension_step6.png)
 
-* Next, create a new password and confirm your password.
+**Step 6:** Create a password that will be used to access your wallet. Ensure you keep this password safe as it will be needed to access your wallet regularly and this password cannot be retrieved. 
 
 ![maiar-defi-example-password](/wallet/wallet-extension/wallet_extension_step7.png)
 
+**Step 7:** Completed! Your Maiar DeFi Wallet has been successfully created and set to be used. 
+
 ![maiar-defi-example-successful](/wallet/wallet-extension/wallet_extension_step8.png)
 
-**Completed successfully!**
 
+### Import existing wallet
 
-## Key features
+Do you already have a wallet?
 
-### Send to a wallet
+Then there is no need to create a new one. The Maiar Wallet Extension provides an option to import your existing wallet. However, to import an existing wallet, you must be able to access the **recovery (secret) phrase** of your wallet. 
 
-With this extension installed, you can send to a wallet from your web browser without having to enter your password, which is a key feature of the Maiar DeFi wallet extension.
+The Maiar wallet has a set of 24-secret phrases, which serve as your wallet’s key. Using your secret phrase to import an existing wallet does not affect your wallet, as all transaction history across all networks will be reflected in the wallet extension.
```

Comment:
I don't understand what you want to describe here. The "import an existing wallet does not affect your wallet" part is confusing.

Reply:
Use the term `words` when referring to the 24 elements of the secret phrase. Also, make sure to only use "secret phrase" when referring to the 24 words (e.g. not "key"). This is important for consistency.

The second sentence of this paragraph, while reassuring to the user, is not that necessary. If you want to assure the user that importing a wallet is safe, you could maybe say that "Using a secret phrase to import an existing wallet does not affect the wallet in any way." Referring to the transaction history may be too much here.